packages/material-ui/src/Switch/Switch.js

```diff
@@ -134,6 +157,7 @@ const Switch = React.forwardRef(function Switch(props, ref) {
         {
           [classes.edgeStart]: edge === 'start',
           [classes.edgeEnd]: edge === 'end',
+          [classes[`size${capitalize(size)}`]]: size !== 'medium',
```

Comment:
```suggestion
          [classes[`size${capitalize(size)}`]]: size === 'small',
```

Reply:
Hold on, I think that we should keep the `size !== 'medium',` so people can, in their theme add more variants.